### DISCUSSION TITLE
头像下面的日志、类别和标签

### DISCUSSION THREAD

**Discussion by zyyaria** [hugo-next/hugo-theme-next#47] [Q&A]
为啥点开是404，是要进行什么操作吗？

**Comment by elkan1788** [comment]
建议可以参考一下 [hugo-next-docs](https://github.com/hugo-next/hugo-next-docs/tree/main/content) 的目录结构，目前需要在 content 目录下创建个名称为 `archives` 子目录，这个是用来显示日志页面的特定链接。这点在 v4.2.0 发布时就已经说明过了：[https://hugo-next.eu.org/hugo-next-v4-2-0-release.html](https://hugo-next.eu.org/hugo-next-v4-2-0-release.html)
